src/google-calendar-api.js

```diff
@@ -57,17 +90,41 @@ module.exports = {
 			)
 		);
 	},
-	createWfhEvent: employeeName => {
+	createWfhEventInInterval: (employeeName, eventStartDateTime, eventEndDateTime) => {
```

Comment:
I think we can add in some "formatting" logic and then this method can basically go away. All this adds is the end date and start time as specific points in time. We could easily use the `!eventEndDateTime` or `eventEndDateTime = eventEndDateTime || defaultDate(startDate)` type logic.